src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -92,6 +91,44 @@ static LoanPolicy from(JsonObject representation) {
     }
   }
 
+  public HttpResult<Loan> overrideRenewal(Loan loan, DateTime systemDate,
+                                          DateTime overrideDueDate, String comment) {
+    try {
+      HttpResult<DateTime> proposedDueDateResult =
+        determineStrategy(true, systemDate).calculateDueDate(loan);
```

Comment:
It would appear that the calculation of the new due date and the validation based upon the loan policy is no longer conducted.

If this the case, I think this comment is redundant now (and can be resolved)? @kostyantyn-kh do you agree?

Reply:
removed dueDate calculation for override logic